### PR TITLE
Chore

### PR DESCRIPTION
**Pull request by verpixelt**
- Remove old comments
- Reorder declarations
- Translate `px` to `rem`
- Get rid of nesting
- Remove empty lines at the beginning of files

### CODE REVIEW

File: _sass/modules/_m-page-footer.scss [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
 .page-footer {
   @extend %sans-serif;
-  font-size: .777777778em;
-  line-height: 1.405864198;;
   background-color: $color-mine-shaft;
+  border-top: .125rem solid $color-dove-gray;
   color: $color-white;
-  border-top: 2px solid $color-dove-gray;
-  padding-top: 2 * $base-unit;
+  font-size: .777777778em;
```
Comment (lukaszklis) [inline _sass/modules/_m-page-footer.scss]: Can we somehow make it more rounded? :D

File: _sass/modules/_m-page-footer.scss [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
+  line-height: 1.405864198;;
```
Comment (lukaszklis) [inline _sass/modules/_m-page-footer.scss]: Same as above

File: _sass/modules/_m-user.scss [hunk 3]
```diff
@@ -36,27 +34,28 @@ $module-namespace: 'user';
...
-  padding: 0 1em 0 0;
   margin: 0;
+  padding: 0 1em 0 0;
 }
+
 .coach__links {
   float: right;
-  padding-left: .5em;
   margin: 0;
+  padding-left: .5em;
```
Comment (lukaszklis) [inline _sass/modules/_m-user.scss]: `rem` instead of `em`? ;)

File: _sass/modules/_m-user.scss [hunk 4]
```diff
@@ -36,27 +34,28 @@ $module-namespace: 'user';
...
 }
 
-// coaches list with leading dots
-
+// COACHES LIST WITH LEADING DOTS
 .coach {
   clear: both;
+
   &:after {
+    border-bottom: 3px dotted $color-silver-chalice;
```
Comment (lukaszklis) [inline _sass/modules/_m-user.scss]: `rem` instead of `px`? :D

File: _sass/modules/_m-page-footer.scss (hunk 1 above)
Comment (lukaszklis) [inline _sass/modules/_m-page-footer.scss]: Also, could we use `rem`s here too? ;)

File: _sass/modules/_m-user.scss (hunk 4 above)
Comment (fredericmarx) [inline _sass/modules/_m-user.scss]: I would stick with `px` for border width values. You typically don’t want a 1px border to become thicker with changing root font size and for small values like this 1px, 3px, etc is more tangible than the `rem`-equivalent.

File: _sass/modules/_m-page-footer.scss (hunk 2 above)
Comment (verpixelt) [inline _sass/modules/_m-page-footer.scss]: Same as above

File: _sass/modules/_m-user.scss (hunk 3 above)
Comment (verpixelt) [inline _sass/modules/_m-user.scss]: will change

File: _sass/modules/_m-user.scss (hunk 4 above)
Comment (verpixelt) [inline _sass/modules/_m-user.scss]: I would usually go with FredFred but in this case I'd say consistency wins. Will change it to `rem`.

**Comment by lukaszklis** [comment]
@verpixelt do you still want to continue working on this PR, or should we close it?

**Pull request closed without merging**